manifest.json

```diff
@@ -22,15 +22,17 @@
       "scripts/background.js"
     ]
   },
-  "browser_action": {
+  "page_action": {
```

Comment:
Does this work on Firefox for Android?

Reply:
According to https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Differences_between_desktop_and_Android#Effect_on_your_add-on_UI, every feature (most likely except `show_matches` and `hide_matches`) required is available on the latest releases.

I have not tested it though, and I'm lacking a proper test environment to do it. If needed, I can set up the thing, and come back with an answer by the end of the week.

Reply:
> Does this work on Firefox for Android?

Yes, now it does, and it looks like this:

![screenshot_2019-01-09-14-37-21](https://user-images.githubusercontent.com/6209647/50912584-c9ea3300-1432-11e9-9988-aa42177ed62c.png)
Screen taken on a 10'' tablet